bluepysnap/simulation.py

```diff
@@ -126,7 +126,7 @@ def simulator(self):
     def node_sets(self):
```

Comment:
Frankly, I am not sure if this should return the `NodeSets` object or the `NodeSets.content`. 

If the use case is resolving `Simulation`'s node sets, then `NodeSets`, if it's just for information, then content, I quess.